<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RLMLogReader</class>
 <widget class="QMainWindow" name="RLMLogReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RLM Log Reader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="inputTextField"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="generateButton">
      <property name="text">
       <string>&amp;Generate</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Select &amp;Directory...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>Browse &amp;File...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string>Report &amp;Output Directory</string>
      </property>
      <property name="buddy">
       <cstring>outputTextField</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="outputTextField"/>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="inputLabel">
      <property name="text">
       <string>RLM &amp;Log File</string>
      </property>
      <property name="buddy">
       <cstring>inputTextField</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionReset_All_Settings">
   <property name="text">
    <string>Reset All Settings</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>inputTextField</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>outputTextField</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>generateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
